docker/config/local_development.env

```diff
@@ -15,3 +15,6 @@ DEBUG=True
 # static files for the webapp get put in /tmp/crashstats-static/ so we
 # need to set STATIC_ROOT to that.
 STATIC_ROOT=/tmp/crashstats-static/
+
+# For connecting to local S3, we need to connect insecurely.
+AWS_SECURE=False
```

Comment:
We require this setting, but since it would be super bad if this ever made it to a server environment by accident, we put it in this file as opposed to `docker/config/docker_common.env`.